### PR TITLE
Copy to Clipboard

### PR DESCRIPTION
**Pull request by miaseiner**
### Description
I made the functionality of the copy to clipboard button.  When you press the button it copies the text in the screenshot below, and you can paste the text wherever you want.

### Issue
#142 

### Screenshot
IMG_0302
IMG_0303

### CODE REVIEW

**Comment by miaseiner** [comment]
Screen Shot 2019-04-24 at 2 28 26 PM
Screen Shot 2019-04-24 at 2 28 35 PM
FINALLY! made names work!!! :))

**Comment by ijemmao** [comment]
@miaseiner Resolve those conflicts and we should be good!

File: screens/AuthorizeDriver.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 import variables from './../assets/data/variables';
-import OrangeBackground from './../components/OrangeBackground';
 import PrimaryButton from '../components/PrimaryButton';
 import Toolbar from '../components/Toolbar';
+import firebase from 'firebase';
+
+
+
+const database = firebase.database();
+    var driverName;
```
Comment (ijemmao) [inline screens/AuthorizeDriver.js]: Make this a state variable

File: screens/AuthorizeDriver.js [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+    var yourFirstName;
```
Comment (ijemmao) [inline screens/AuthorizeDriver.js]: Make this a state variable

File: screens/AuthorizeDriver.js [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
+    var yourLastName;
```
Comment (ijemmao) [inline screens/AuthorizeDriver.js]: Make this a state variable

File: screens/AuthorizeDriver.js [hunk 4]
```diff
@@ -15,17 +23,25 @@ export default class AuthorizeDriver extends React.Component {
     }
 
     copyEmail = () => {
+        const props = this.props;
+        database.ref('/deliveries/' + 'delivery'+this.props.navigation.getParam('deliveryKey')).once('value').then(function(snapshot) {
```
Comment (ijemmao) [inline screens/AuthorizeDriver.js]: Use backward apostrophes notion

File: screens/AuthorizeDriver.js [hunk 5]
```diff
@@ -15,17 +23,25 @@ export default class AuthorizeDriver extends React.Component {
...
+                driverName = snapshot.val().driver;
+            database.ref('/users/' + props.navigation.state.params['userKey']).once('value').then(function(snapshot) {
```
Comment (ijemmao) [inline screens/AuthorizeDriver.js]: Use backward apostrophes notion

**Pull request merged**